tools/merge_duplicates.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+from database.FirebaseAccessor import FirebaseAccessor
+import asyncio
+from argparse import ArgumentParser
+
+class CLIParser(ArgumentParser):
+    def __init__(self, *kargs, **kwargs):
+        super().__init__(*kargs, **kwargs)
+        self.add_argument("--strategy", choices=["latest"], default="latest")
+
+def get_merger(strategy):
+    if strategy == 'latest':
```

Comment:
There is likely some value in also creating the merger to always select the oldest, given the oldest is usually the most likely to have additional metadata attached to it